source/specifications.rst

```diff
@@ -94,17 +94,12 @@ be ``UTF-8``.
 
 Other parameters might be specific to the chosen subtype. For example, for the
 ``markdown`` subtype, there is a ``variant`` parameter that allows specifying
-the variant of Markdown in use, such as:
+the variant of Markdown in use. Currently, the only value that is recognized
```

Comment:
I guess optional makes sense since there's only one variant we're supporting? I dunno, I don't feel super strongly about it.